packages/amplify-graphql-model-transformer/src/__tests__/model-transformer.test.ts

```diff
@@ -1255,6 +1255,26 @@ describe('ModelTransformer: ', () => {
         },
       });
   });
+
+  it('the datastore table TTL spec should not be removed but disabled when datastore is disabled', () => {
```

Comment:
Could you add a positive test case as well? I believe it's covered in snapshots, but would prefer to get towards these explicit tests.

Reply:
Added one